cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/system/SystemWorkerHttpHandlerInternal.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
```

Comment:
nit: Please change the copyright year to 2022, thanks! Same for all files.

Reply:
Done.